ios/FluentUI/Core/FluentUIFramework.swift

```diff
@@ -102,28 +87,31 @@ public class FluentUIFramework: NSObject {
 
         let progressViewAppearance = containerTypes != nil ? UIProgressView.appearance(whenContainedInInstancesOf: containerTypes!) : UIProgressView.appearance()
         progressViewAppearance.progressTintColor = primaryColor
-        progressViewAppearance.trackTintColor = Colors.Progress.trackTint
+        progressViewAppearance.trackTintColor = UIColor(dynamicColor: aliasTokens.colors[.stroke1])
     }
 
     static func initializeUINavigationBarAppearance(_ navigationBar: UINavigationBar, traits: UITraitCollection? = nil, navigationBarStyle: NavigationBarStyle = .normal, fluentTheme: FluentTheme? = nil) {
         navigationBar.isTranslucent = false
 
         let standardAppearance = navigationBar.standardAppearance
-        navigationBar.tintColor = Colors.NavigationBar.tint
-
-        if let fluentTheme = fluentTheme {
-            navigationBar.standardAppearance.backgroundColor = navigationBarStyle.backgroundColor(fluentTheme: fluentTheme)
-        } else {
-            navigationBar.standardAppearance.backgroundColor = Colors.NavigationBar.background
-        }
+        let aliasTokens = AliasTokens()
```

Comment:
Why is line 74 `let aliasTokens = fluentTheme?.aliasTokens ?? AliasTokens()` but here we default to a new instance of `AliasTokens()`?

Reply:
I believe I've resolved this already :)